### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by claus66** [vinaypamnani/wmie2#5]
Could you please add user+pwd to command line option ie: 
wmiexplorer computer1 domain\user pwd

**Comment by LUCScape** [comment]
Don't want passwords in command lines that are logged in event log.  Have computer and userid on command line, but require user to enter the password into GUI.

**Comment by vinaypamnani** [comment]
Yeah, I wouldn't want passwords specified in the command line either, but adding the username and prompting for password is do-able. I'll add this in future when I work on this again.

**Comment by vinaypamnani** [comment]
Try the new version https://github.com/vinaypamnani/wmie - it allows specifying computer/username as params but password requires you to input within app for secure handling.
